src/node/progress_tracker.h

```diff
@@ -525,12 +563,102 @@ namespace ccf
       return highest_commit_level;
     }
 
+    std::unique_ptr<ViewChange> get_view_change_message(
+      kv::Consensus::View view)
+    {
+      auto it = certificates.find(highest_prepared_level.version);
+      if (it == certificates.end())
+      {
+        throw ccf::ccf_logic_error(fmt::format(
+          "Invalid prepared level, view:{}, seqno:{}",
+          highest_prepared_level.term,
```

Comment:
term?

Reply:
We are standardizing on size right?

Reply:
Yes, let's do that. As discussed, get_view() in the KV is going away, so the opportunity for confusion should be temporary at most.